straxen/plugins/nveto_recorder.py

```diff
@@ -16,7 +16,8 @@
                  help="Pretrigger time before coincidence window in ns."),
     strax.Option('resolving_time_recorder_nv', type=int, default=600,
                  help="Resolving time of the coincidence in ns."),
-    strax.Option('nbaseline_samples_lone_records_nv', type=int, default=10, track=False,
+    strax.Option('nbaseline_samples_lone_records_nv', type=int,
+                 default_by_run=[(0, 10), (12684, 26)], track=True,
```

Comment:
Is this some git thing or did you already change this before? (I thought you did or am I now confused with a different plugin).

Reply:
I changed it for records, but forgot to change it here. "lone_records" are thrown at this stage, but in order to compute some properties e.g. like average area per pulse I have to do the whole baseline business here too.